nebula/client.go

```diff
@@ -0,0 +1,458 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
```

Comment:
Why do we want to move client.go to the nebula directory? Here we use go modules, so we don't need one more directory.

Reply:
When we import `nebula-go`, we should always alias to `nebula` like this:
``` 
import (
   nebula "github.com/vesoft-inc/nebula-go"
)
```
Just a little wordy